doc/source/guide/data_types/spectra.rst

```diff
@@ -97,7 +97,7 @@ To get rid of the noise, we could also clip low intensities by setting vmin.
 
 .. plot::
 
-    from matplotlib import pyplot as plt
+    import matplotlib.pyplot as plt
     import sunpy
     import sunpy.data.sample
     from sunpy.spectra.sources.callisto import CallistoSpectrogram
```

Comment:
@dpshelio @segfaulthunter You have any idea why this example would be trying to access the internet? At least I think it's this one.